cmd/deploy/deploy.go

```diff
@@ -73,10 +73,8 @@ type Options struct {
 	Name             string
 	Namespace        string
 	K8sContext       string
-	Repository       string
-	Branch           string
 	Variables        []string
-	servicesToDeploy []string
+	ServicesToDeploy []string
```

Comment:
I think we could revert this. It wouldn't be needed anymore as we don't include the flag in the test command anymore

Reply:
This would make `servicesToDeploy` the only private member. Based on current feedback we could also make `Wait` and `Dependencies` also private but that would leave us with some private options and some public options driven by external requirements. Given that `Options` is public, I would keep every member public for simplicity.